java/http/java-http-consumer/src/main/java/HttpConsumer.java

```diff
@@ -61,6 +80,8 @@ public static void main(String[] args) throws URISyntaxException, IOException, I
         }
 
         HttpConsumer consumer = new HttpConsumer(config, messagesReceivedLatch);
+        enableAutoCommit = (props.getProperty(ENABLE_AUTO_COMMIT_CONFIG)) == null || Boolean.parseBoolean(props.getProperty(ENABLE_AUTO_COMMIT_CONFIG));
+
```

Comment:
What's the purpose of setting the enableAutoCommit here? You are already good by reading it in the consumer creation method so you can make this field non static and remove the suppress warnings. See my other comments.

Reply:
Removed